Fix all checkstyle issues.

MaxGroenenboom opened the pull request:
Fixed all checkstyle issues.
Fixed all warnings.
Changed indentation for `Function`, `ParameterList` and other lists of `Parameter` constructions to enhance readability.

Travis: [![Build Status](https://travis-ci.org/Denpeer/tygron.svg?branch=checkstyle_fix_all)](https://travis-ci.org/Denpeer/tygron)

MaxGroenenboom committed:
Merge branch 'context' of https://github.com/eishub/tygron into checkstyle_fix_all

MaxGroenenboom committed:
Fixed all checkstyle issues.

Also removed all warnings and changed to alternative indentation for Function and ParameterList construction for better readability.

JoshCode commented:
Build passes, so approved.

\- CodeFox [![CF](http://i.imgur.com/dyOqeMH.png)](http://github.com/CodeFoxNL)

HarmGriffioen commented:
Merge conflicts, please resolve them

MaxGroenenboom commented:
Yeah those were introduced by merging the other PR's...

MaxGroenenboom committed:
Merge branch 'context' of https://github.com/eishub/tygron into checkstyle_fix_all

MaxGroenenboom commented:
So I got a failing test now: `tranlatorTest1(contextvh.translators.J2UpgradeTypeTest): No translator found for class nl.tytech.data.engine.serializable.UpgradePair$$EnhancerByMockitoWithCGLIB$$75fc3dd8` (with a typo in the name of the test btw), while I obviously didn't change any functional code...
Also I did test locally and it succeeded :/

MaxGroenenboom committed:
Add a J2UpgradePair to the translator used by J2UpgradeType.

Without this fix, junit apparently fails on J2UpgradeTypeTest because a translator isn't added.
I don't know why this doesn't happen in other tests, please verify me and Travis aren't doing something wrong.

MaxGroenenboom committed:
Removed typo in test name.

J2UpgradeTypeTest.tranlatorTest1 -> J2UpgradeTypeTest.translatorTest1.

MaxGroenenboom commented on contextvh/src/main/java/contextvh/translators/J2UpgradeType.java:
```diff
@@ -18,6 +18,13 @@
+	 * Adds a translator for J2UpgradePair if it isn't added yet.
+	 */
+	public J2UpgradeType() {
+		translator.registerJava2ParameterTranslator(new J2UpgradePair());
```
Please verify this is necessary, I haven't seen this in other Translator classes that use other Translators, but for some reason the test for this class was failing.

`tranlatorTest1(contextvh.translators.J2UpgradeTypeTest): No translator found for class nl.tytech.data.engine.serializable.UpgradePair$$EnhancerByMockitoWithCGLIB$$75fc3dd8`

MaxGroenenboom commented:
Please verify f6816f1 is necessary, I haven't seen this construction in other Translator classes that use other Translators, but for some reason the test for this class was failing.

`tranlatorTest1(contextvh.translators.J2UpgradeTypeTest): No translator found for class nl.tytech.data.engine.serializable.UpgradePair$$EnhancerByMockitoWithCGLIB$$75fc3dd8`

MaxGroenenboom commented:
Travis builds now so I hope this can be reviewed and merged quickly.

levilime commented:
So this is a refactor because of the checkstyle change in #87? A member of team Danshal was okay with not having any trailing spaces so we'll see this as part of that merge too  and agree with doing away with all spaces in the code. Also the added comments is a nice touch.

:star:

MaxGroenenboom commented:
Yeah I figured it was now possible to just checkstyle the entire module to make it easier to check for checkstyle issues.

spiderbiggen commented:
@MaxGroenenboom for the problem with J2UpgradeType test it seems that test is also missing the 3rd test. It seems the changes to that test were lost in a merge

MaxGroenenboom commented:
@spiderbiggen I didn't remove it as you can see in the diff...
https://github.com/eishub/tygron/pull/93/files#diff-bde1517c676bee363d4351b115a5b107L16

spiderbiggen commented:
I know but it is lost somewhere in a merge

spiderbiggen commented:
ah found it

MaxGroenenboom commented:
@spiderbiggen according to this history (https://github.com/eishub/tygron/commits/context/contextvh/src/test/java/contextvh/translators/J2UpgradeTypeTest.java) it was removed by a revert of a PR from @klikooo. If I remember right he reverted a PR that caused the context branch to fail on travis.

MaxGroenenboom commented:
Here is the revert PR I mentioned in the last comment: #89

spiderbiggen commented:
I will create a fix

spiderbiggen commented:
all right I will do that after this is merged as it now always breaks on checkstyle
we approve of this PR

RAMpestampers
